<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentShowDialog</class>
 <widget class="QDialog" name="EquipmentShowDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>581</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sortByMatchIdButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>380</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按设备编号进行排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortByNameButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>380</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按地区进行排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sortByAreaButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按设备名称进行排序</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
  </widget>
  <widget class="QLabel" name="searchLabel">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>80</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>查询区</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="namelineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="nameButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>140</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按设备名称查询</string>
   </property>
  </widget>
  <widget class="QPushButton" name="matchIdButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>200</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按设备编号查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="matchIdlineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="areaButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>260</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>按地区查询</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="arealineEdit">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>260</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
